laterpay/application/Controller/Frontend/Post.php

```diff
@@ -105,6 +113,17 @@ public function ajax_redeem_voucher_code( LaterPay_Core_Event $event ) {
         $code       = sanitize_text_field( $_GET['code'] ); // phpcs:ignore
         $code_data  = LaterPay_Helper_Voucher::check_voucher_code( $code );
         $lp_post_id = intval( $_GET['lp_post_id'] ); // phpcs:ignore
+        $type       = filter_input( INPUT_GET, 'type', FILTER_SANITIZE_STRING );
+        $type       = sanitize_text_field( $type );
+
+        $requested_pass_id = filter_input( INPUT_GET, 'pass_id', FILTER_SANITIZE_STRING );
+        $requested_pass_id = ( ! empty( $requested_pass_id ) && 0 < intval( $requested_pass_id ) ) ? absint( $requested_pass_id ) : 0;
```

Comment:
```suggestion
        $requested_pass_id = ( ! empty( $requested_pass_id ) && 0 < absint( $requested_pass_id ) ) ? absint( $requested_pass_id ) : 0;
```

Reply:
I think we should use `intval()` it self. Since `absint()` will make value absolute (-1 will be 1) that we shouldn't allow.